build.gradle

```diff
@@ -17,6 +17,7 @@ buildscript {
         version_qualifier = System.getProperty("build.version_qualifier", "")
         opensearch_group = "org.opensearch"
         isSnapshot = "true" == System.getProperty("build.snapshot", "true")
+        enable_SIMD = System.getProperty("enable_SIMD", "false")
```

Comment:
Could we follow convention of security.enabled as -Dsimd.enabled?

Reply:
updated